lib/core/cardano-wallet-core.cabal

```diff
@@ -60,6 +60,7 @@ library
     , persistent
     , persistent-sqlite
     , persistent-template
+    , QuickCheck
```

Comment:
That is very regrettable though :s ... We kept our core code free of any test assets so far an, I admit that introducing a QC dependency at this level makes me particularly sad / anxious.

Reply:
This is a fair point.

I've moved the instance to a separate package `test-utils` that can be imported just by targets related to testing.